tests/integration/test_tls.py

```diff
@@ -29,11 +29,19 @@
 APP_NAME = METADATA["name"]
 
 if juju_.has_secrets:
-    TLS_APP_NAME = "self-signed-certificates"
-    TLS_CONFIG = {"ca-common-name": "Test CA"}
+    tls_app_name = "self-signed-certificates"
+    if architecture.architecture == "arm64":
```

Comment:
do we need a TODO here to use `latest/stable` once arm64 has been promoted to stable for the certificates operator?